Telemetry/CorePing.swift

```diff
@@ -64,6 +69,13 @@ class CorePing: TelemetryPing {
 
         let timezoneOffset = NSTimeZone.localTimeZone().secondsFromGMT / 60
 
+        let usageCount = UsageTelemetry.getCount(prefs)
+        let usageTime = UsageTelemetry.getTime(prefs)
+        UsageTelemetry.reset(prefs)
+
+        profile.prefs.setInt(Int32(usageTime), forKey: PrefKeyUsageTime)
+        profile.prefs.setInt(Int32(usageCount), forKey: PrefKeyUsageTime)
```

Comment:
What are these used for? I don't seem them used anywhere.

Also, I assume this line should be `PrefKeyUsageCount`.